src/App.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+class App extends Component {
+  render() {
+    return (
+      <div className="App">
+        <header className="App-header">
+          <img src={logo} className="App-logo" alt="logo" />
+          <h1 className="App-title">tic tac toe</h1>
+        </header>
+      </div>
```

Comment:
We are missing one thing here.

Please note that in this file, our only job is to define and export the `App` component. We're defining it here, and exporting it at the end of the file.

This is fine, but note that we're also defining several other components. The main one is the `Game` component. `Game` uses `Board`, and `Board` uses `Square`.

The `Game` component, using those subcomponents, is able to render a nice tic-tac-toe game.

The issue is, we're not using `Game` anywhere!

Components are like HTML tags. But you can define your own. We just defined the `Game` component, that could be used like  this: `<Game />`. But we're never using it!

We know that our `App` component _is_ being used, so you can add `<Game />` here, inside the `render()` method of `App`, and you'll see a tic tac toe board!